src/utils/helpers/focus-trap/focus-trap.spec.js

```diff
@@ -28,6 +30,51 @@ describe('focusTrap', () => {
   const shiftTabKey = new KeyboardEvent('keydown', { keyCode: 9, shiftKey: true });
   const otherKey = new KeyboardEvent('keydown', { keyCode: 32 });
 
+  describe('and element has callback function for focus', () => {
+    let wrapper, onFocus;
+
+    beforeEach(() => {
+      onFocus = jest.fn();
+      wrapper = mount(
+        <TestComponent focusFirstElement={ onFocus }>
+          <button type='button'>Test button One</button>
+          <button type='button'>Test button Two</button>
+        </TestComponent>,
+        { attachTo: htmlElement }
+      );
+    });
+
+    afterEach(() => {
+      wrapper.unmount();
+    });
+
+    it('should focus second focusable item', () => {
```

Comment:
wrong description? `should focus first focusable item` ?

Reply:
yes, updated. thanks!